### DISCUSSION TITLE
スレタイの先頭に半角スペースが入るようになりました

### DISCUSSION THREAD

**Discussion by Opus0159** [edginer/eddiner#44] [General]
最近Sikiでエッヂにスレを立てるとスレタイの先頭に半角スペースが1個入るようになりました。
おま環か？とも思いエッヂで他の人に聞くとちんくるでも同じ状態のようです。
スレタイをコピーする時等に弊害があるのでスペースが入らないようにしてほしいです。

**Comment by Opus0159** [comment]
昨日対応していただいたようで先頭スペースが消えました。ありがとうございます。
ただ、liveなスレはスペースがなくなったのですが、そのスレがdat落ちするとスペースが付いてしまうようです。

**Comment by edginer** [comment]
対応しました
